small_lms_test.py

```diff
@@ -12,16 +14,18 @@
 
 from model import BigramModel
 # -----------------------------
-
 # setup cuda
-torch.cuda.memory._record_memory_history()
-device = "cuda" if torch.cuda.is_available() else "cpu"
-torch.set_default_device("cpu")
+if torch.cuda.is_available():
+    torch.cuda.memory._record_memory_history()
+    device = "cuda"
+else: device="cpu"
+torch.set_default_device(device)
```

Comment:
In test it's not an issue because we are creating tensors on the relevant device explicitly, but still remove for consistency.